service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -310,6 +310,46 @@ fi
 # integration test will ensure that everything in script worked.
 sudo -u "$JUPYTER_USER" sh -c 'terra git clone --all'
 
+################################
+# Setup instance startup service
+################################
+# Create a script to perform the following steps on instance startup:
```

Comment:
Maybe to make this more clear, "on each instance reboot"?  Or just in general make it clear in the comment that we have this script to run on every reboot, not just one-time?